server/models/PetSitter.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+const Profile = require("./Profile");
+
+const petSitterSchema = new mongoose.Schema({
+  stripeConnectId: {
+    type: String,
+    required: true,
+  },
+  availabilityId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "Availability",
+  },
+  activatedAvailabilitySchedule: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "Schedule",
+  },
+  rate: {
+    type: Number,
+    required: true,
+  },
```

Comment:
It would be cool to validate that this is a "reasonable" number! ;)

Reply:
I have changed something. Please let me know if that is what we need!